lib/awesome_nested_set/model.rb

```diff
@@ -95,6 +99,12 @@ def root
           def roots
             nested_set_scope.children_of nil
           end
+
+          private
+
+          def scope_order_from_options(options)
+            options[:order] || { order_column_name => :asc }
```

Comment:
Maybe we should use fetch:

```suggestion
            options.fetch(:order) { { order_column_name => :asc } }
```

Reply:
I assume you mean ?
```
options.fetch(:order, { { order_column_name => :asc }) 
```

It seems about the same thing

Reply:
Yeah, fetch takes a second argument, or a block so it can be 
```ruby
options.fetch(:order) { { order_column_name => :asc } }
```

or it can be
```ruby
options.fetch(:order, { order_column_name => :asc })
```

Whatever is your preference 👍